src/arch/host/include/arch/spinlock.h

```diff
@@ -10,12 +10,12 @@
 #ifndef __ARCH_SPINLOCK_H__
 #define __ARCH_SPINLOCK_H__
 
-typedef struct {
-} spinlock_t;
+struct spinlock_s {
+};
 
```

Comment:
Why spinlock_s? I think just spinlock it's fine.

Reply:
I did it as suggested by @lyakh . I'm also not sure if doing so many back-and-forth changes is that productive, so I think I'll stick with this unless someone else agrees with you.

Reply:
@paulstelian97 all right, no problem.

Reply:
@dbaluta for the same reason why spinlock_t. Maybe it's slightly less often used, but it's often / sometimes used in such situations. It's by no means a compulsory convention, so `struct spinlock` or `struct _spinlock` or `struct spin_lock` or anything else could be used too. Just not `struct _spinlock_t` because `_t` is really reserved for `typedef` types

Reply:
@lyakh _t from spinlock_t is there because that is a typedef as you say. It doesn't make much sense
to add _s for structures because struct should be always present in 'struct spinlock'

Reply:
@dbaluta you're right, `struct spinlock` would've been better, I just wasn't sure if we already had such a struct.

Reply:
Changed to `struct spinlock`. Builds fine.